### ISSUE TITLE
confused by loss function forward params

### ISSUE THREAD

**Issue by cswwp** [qubvel/segmentation_models#188]
Hi qubvel, i am confused by the i was confused by loss function forward params.

def iou(pr, gt, eps=1e-7, threshold=None, activation='sigmoid'):
    """
    Source:
        https://github.com/catalyst-team/catalyst/
    Args:
        pr (torch.Tensor): A list of predicted elements
        gt (torch.Tensor):  A list of elements that are to be predicted
        eps (float): epsilon to avoid zero division
        threshold: threshold for outputs binarization
    Returns:
        float: IoU (Jaccard) score
    """

def f_score(pr, gt, beta=1, eps=1e-7, threshold=None, activation='sigmoid'):
    """
    Args:
        pr (torch.Tensor): A list of predicted elements
        gt (torch.Tensor):  A list of elements that are to be predicted
        beta (float): positive constant
        eps (float): epsilon to avoid zero division
        threshold: threshold for outputs binarization
    Returns:
        float: F score
    """
As shows,  pr (torch.Tensor)  is a tensor with shape[batch, channel, width, height]?  but why write "A list of predicted elements" as follows, i'm confused with what is the pr, tensor with shape[batch, channel, width, height] or list with element num batch * channel * width * height, which one?

**Comment by qubvel** [comment]
Hi @cswwp 
1) This is Keras repo (issues related to pytorch models should be opened in another repo)
2) Please provide permalink to lines of code, where you consider an error
Thank you.
